src/components/Common/Pipe.css

```diff
@@ -0,0 +1,10 @@
+.Pipe {
+  width: 20px;
+  height: 0;
+  left: 755px;
```

Comment:
Are these left and top styles doing anything? Looks weird without an absolute positioning and without context